services/ui-src/src/components/fields/ChoiceListField.tsx

```diff
@@ -171,18 +144,52 @@ export const ChoiceListField = ({
     }
   };
 
-  // update field values
+  const getNestedChildFieldsOfUncheckedParent = (choices: FieldChoice[]) => {
+    // set up nested field compilation
+    const nestedFields: any = [];
+    const compileNestedFields = (fields: FormField[]) => {
+      fields.forEach((field: FormField) => {
+        // for each child field, get field info
+        const fieldDefaultValue = ["radio", "checkbox"].includes(field.type)
+          ? null
+          : "";
+        const fieldInfo = {
+          name: field.id,
+          shouldClear: true,
+          defaultValue: fieldDefaultValue,
+        };
+        // add to nested fields to be autosaved
+        nestedFields.push(fieldInfo);
+        // recurse through additional nested children as needed
+        const nestedChildren = field?.props?.choices;
+        if (nestedChildren) compileNestedFields(nestedChildren);
+      });
+    };
+
+    choices.forEach((choice: FieldChoice) => {
+      // if choice is not selected and there are children
+      if (!choice.checked && choice.children) {
+        compileNestedFields(choice.children);
+      }
+    });
+
+    return nestedFields; // TODO: just map here?
+  };
+
+  // if should autosave, submit field data to database on component blur
   const onComponentBlurHandler = async () => {
-    const willAutosave = shouldAutosave(
-      displayValue,
-      lastAutosaveValue,
-      autosave,
-      userIsStateRep,
-      userIsStateUser
-    );
-    if (willAutosave) {
-      setLastAutosaveValue(displayValue);
-      autosaveSelections(displayValue);
+    if (autosave) {
+      let fields = [
+        { name, value: displayValue, hydrationValue, defaultValue },
+        ...getNestedChildFieldsOfUncheckedParent(choices),
+      ];
```

Comment:
Note: Slightly different than other fields because we have to pass all the nested children of the unchecked parent.